### PR TITLE
Remove hard dependency on FSiDatagrid, update tests and Travis suite

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,16 +12,16 @@
...
         "symfony/twig-bundle": "^2.4|^3.0",
         "symfony/http-foundation": "^2.4|^3.0",
         "symfony/options-resolver": "^2.6|^3.0",
         "symfony/form" : "^2.4|^3.0",
         "symfony/validator": "^2.4|^3.0",
-        "doctrine/doctrine-bundle": "~1.2",
-        "fsi/doctrine-extensions": "~1.1,>=1.1.4",
-        "sensio/framework-extra-bundle": "~3.0",
+        "doctrine/doctrine-bundle": "^1.2",
+        "fsi/doctrine-extensions": "^1.1,>=1.1.4",
```
Comment (chives) [inline composer.json]: `^1.1.4`

**Comment by szymach** [comment]
@fsi-open thoughts?

**Comment by chives** [comment]
compare [this solution](https://github.com/fsi-open/datagrid-bundle/commit/f9de39d466d550d781ce1c8e7c74138e91e30ae8) for sf3.1 bc break

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
Issue:
1. The services cannot be loaded in the extension class - it only receives a copy of the container, without the actual list of other extensions.
2. They cannot be loaded in the compiler pass, because it is ran after the Datagrid Bundle's pass, which actually does the loading of tagged services.
3. It can be added in the bundle class, since it has access to all extensions and is done before compiler passes are ran. Seems kinda wrong, though.

Any ideas?

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open change as suggested by @chives

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@rn0

**Comment by szymach** [comment]
@rn0

**Comment by szymach** [comment]
@rn0

**Pull request merged**